File: InventoryAPI/InventoryAPI/Models/Supplier.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace InventoryAPI.Models
{
    public partial class Supplier
    {
        public long Id { get; set; }
        public string Name { get; set; }
        public string Contact { get; set; }
        public string Address { get; set; }
    }
}

File: InventoryApp/src/app/models/supplier.ts
export class Supplier {
    id: number;
    name: string;
    contact: string;
    address: string;
    static fromJsonArray(jsonArray): Supplier[] {
      const suppliers = [];
  
      jsonArray.forEach(element => {
        const supplier = new Supplier();
        supplier.id = element.id;
        supplier.name = element.name;
        supplier.contact = element.contact;
        supplier.address = element.address;
        suppliers.push(supplier);
      });
  
      return suppliers;
    }
  }